File: estomagordo-python3/2a.py
def solver(matrix):
    return sum(max(row) - min(row) for row in matrix)

with open('input_2.txt', 'r') as f:    
    matrix = []
    for line in f.readlines():
        matrix.append(list(map(int, line.split())))
    print(solver(matrix))

File: telcopro-go/2/2a.go
package main

import "math"

func rowdiff(row [16]int) int {

	var min, max int = math.MaxInt32, 0

	for _, i := range row {
		if i < min {
			min = i
		}
		if i > max {
			max = i
		}
	}
	return (max - min)
}
